stdlib/xml/dom/minidom.pyi

```diff
@@ -265,10 +265,10 @@ class Notation(Identified, Childless, Node):
     def __init__(self, name, publicId, systemId) -> None: ...
 
 class DOMImplementation(DOMImplementationLS):
-    def hasFeature(self, feature, version) -> bool: ...
-    def createDocument(self, namespaceURI: str | None, qualifiedName: str | None, doctype): ...
-    def createDocumentType(self, qualifiedName: str | None, publicId, systemId): ...
-    def getInterface(self, feature): ...
+    def hasFeature(self, feature: str, version: str | None) -> bool: ...
+    def createDocument(self, namespaceURI: str | None, qualifiedName: str | None, doctype: DocumentType | None) -> Document: ...
+    def createDocumentType(self, qualifiedName: str | None, publicId: Incomplete, systemId: Incomplete) -> Incomplete: ...
```

Comment:
According to the documentation for this function, `publicId` and `systemId` should both be strings: https://docs.python.org/3/library/xml.dom.html#xml.dom.DOMImplementation.createDocumentType

And we can see from the source that this function returns an instance of `DocumentType`: https://github.com/python/cpython/blob/e0d54a4a799dae4ebdd72a16bcf287ed62ae2972/Lib/xml/dom/minidom.py#L1477

```suggestion
    def createDocumentType(self, qualifiedName: str | None, publicId: str, systemId: str) -> DocumentType: ...
```

Reply:
Ah, I must have missed that. Thanks for catching this 👍